ft-email-auth-provider/src/handlers/create_account.rs

```diff
@@ -16,31 +17,41 @@ impl CreateAccount {
             .timestamp_nanos_opt()
             .expect("unexpected out of range datetime");
 
-        ft_sdk::auth::ProviderData {
+        let mut res = ft_sdk::auth::ProviderData {
             #[cfg(feature = "username")]
             identity: self.username.to_string(),
             #[cfg(not(feature = "username"))]
-            identity: self.email.to_string(),
+            identity: Some(self.email.to_string()),
```

Comment:
The identity type is optional for one feature and not optional for other. This will fail. Try running `cargo check --no-default-features`

Reply:
done in https://github.com/fastn-community/auth/pull/7/commits/ad2940ff8fee2deb49e49d6a4aafafbc07df03ac